src/views/admin/donates.hbs

```diff
@@ -0,0 +1,34 @@
+<main class="main">
+  <section class="taps">
+    <input type="button" value="New Donations" class="new">
+    <input type="button" value="Current Donations" class="current">
+    <input type="button" value="Done Donations" class="done">
+  </section>
+  {{#each card}}
+  {{#unless card.state}}
```

Comment:
@AbdallahAmmar96 You might need `this.state` to access the state of the current card in the `#each` loop